### PR TITLE
II-456 Use kubernetes recommended labels and latest chart templates

### PR DESCRIPTION
**Pull request by bkompis**
https://v1-14.docs.kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1517/ : SUCCESS in 10s

File: k8s/charts/lcm-bricks/templates/prometheus/alertingRules.yaml [hunk 1]
```diff
@@ -4,11 +4,12 @@ metadata:
...
   namespace: monitoring
   labels:
-    app: lcm-bricks
-    component: lcm
+    app.kubernetes.io/name: lcm-bricks
+    app.kubernetes.io/component: lcm
     team: lcm
-    chart: lcm-bricks
-    release: {{ .Release.Name }}
+    helm.sh/chart: lcm-bricks
```
Comment (jakub-plichta) [inline k8s/charts/lcm-bricks/templates/prometheus/alertingRules.yaml]: should contain version

File: k8s/charts/lcm-bricks/templates/prometheus/alertingRules.yaml [hunk 2]
```diff
@@ -4,11 +4,12 @@ metadata:
   name: lcm-bricks-monitoring-rules
   namespace: monitoring
   labels:
-    app: lcm-bricks
-    component: lcm
+    app.kubernetes.io/name: lcm-bricks
```
Comment (jakub-plichta) [inline k8s/charts/lcm-bricks/templates/prometheus/alertingRules.yaml]: should be templated

File: k8s/charts/lcm-bricks/templates/prometheus/alertingRules.yaml [hunk 3]
```diff
@@ -4,11 +4,12 @@ metadata:
...
+    app.kubernetes.io/component: lcm
```
Comment (jakub-plichta) [inline k8s/charts/lcm-bricks/templates/prometheus/alertingRules.yaml]: this doesn't make sense.. should this be `part-of` label instead?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1518/ : SUCCESS in 25s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1529/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-check-pipeline/18/ : SUCCESS in 1m 32s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1531/ : SUCCESS in 19s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/185/ : SUCCESS in 12m 36s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/319/ : SUCCESS in 54s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-travis-check/39/ : FAILURE in 48s (non-voting)
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-deploy-pipeline/10/ : SUCCESS in 2m 14s

**Pull request merged**